sample/host/package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "host",
+    "author": "Workleap",
+    "version": "0.0.0",
+    "description": "Host application to showcase @squide.",
+    "license": "Apache-2.0",
+    "type": "module",
+    "scripts": {
+        "dev": "webpack serve --config webpack.dev.js"
+    },
+    "devDependencies": {
+        "@pmmmwh/react-refresh-webpack-plugin": "0.5.10",
+        "@swc/core": "1.3.56",
+        "@types/react": "18.2.0",
+        "@types/react-dom": "18.2.0",
+        "@types/webpack": "5.28.0",
+        "@workleap/eslint-plugin": "1.2.0",
+        "@workleap/typescript-configs": "2.1.0",
+        "html-webpack-plugin": "5.5.1",
+        "react": "18.2.0",
+        "react-dom": "18.2.0",
+        "react-router": "6.11.1",
+        "react-router-dom": "6.11.1",
+        "swc-loader": "0.2.3",
+        "webpack": "5.82.0",
+        "webpack-cli": "5.0.2",
+        "webpack-dev-server": "4.13.3"
+    },
+    "dependencies": {
+        "@squide/core": "workspace:*",
```

Comment:
squid core is not necessery since everything is reexported by squide/react-router.

If we want client to install the squide core, it should be a peer dependency of @squide/react-router